app/assets/stylesheets/vendor/font_awesome/_icons.scss

```diff
@@ -5,7 +5,7 @@
 .#{$fa-css-prefix}-music:before { content: $fa-var-music; }
 .#{$fa-css-prefix}-search:before { content: $fa-var-search; }
 .#{$fa-css-prefix}-envelope-o:before { content: $fa-var-envelope-o; }
-.#{$fa-css-prefix}-heart:before { content: $fa-var-heart; }
+.#{$fa-css-prefix}-heart:before { content: $fa-var-heart; color: $love; }
```

Comment:
I wouldn't make the change in this file, because it's going to be overwritten on a Font Awesome upgrade. Put it in one of the `common` stylesheets.